core/src/components/searchbar/searchbar.tsx

```diff
@@ -304,13 +359,18 @@ export class Searchbar implements ComponentInterface {
     this.focused = false;
     this.ionBlur.emit();
     this.positionElements();
+
+    if (this.focusedValue !== this.value) {
+      this.emitValueChange();
+    }
```

Comment:
Why do we not clear `this.focusedValue`? Input does this: https://github.com/ionic-team/ionic-framework/blob/feature-7.0/core/src/components/input/input.tsx#L397

Reply:
I'm not 100% sure it's needed but I'll add it in to account for any edge cases I'm missing 👀